Source/Fuse.Scripting.JavaScript/V8/V8Simple.cpp.uxl

```diff
@@ -8,12 +8,12 @@
 	<!-- Build V8Simple on Android, and link static V8 -->
 	<CopyFile Condition="ANDROID" SourceFile="include/V8Simple.cpp" />
 	<Require Condition="ANDROID" IncludeDirectory="@('android/include':Path)" />
-	<Require Condition="ANDROID" StaticLibrary="@('android/lib/libv8_init.a':Path)" />
-	<Require Condition="ANDROID" StaticLibrary="@('android/lib/libv8_initializers.a':Path)" />
-	<Require Condition="ANDROID" StaticLibrary="@('android/lib/libv8_libbase.a':Path)" />
-	<Require Condition="ANDROID" StaticLibrary="@('android/lib/libv8_libplatform.a':Path)" />
-	<Require Condition="ANDROID" StaticLibrary="@('android/lib/libv8_libsampler.a':Path)" />
-	<Require Condition="ANDROID" StaticLibrary="@('android/lib/libv8_snapshot.a':Path)" />
+	<Require Condition="ANDROID" StaticLibrary="@('android/lib/@(ABI)/libv8_init.a':Path)" />
+	<Require Condition="ANDROID" StaticLibrary="@('android/lib/@(ABI)/libv8_initializers.a':Path)" />
+	<Require Condition="ANDROID" StaticLibrary="@('android/lib/@(ABI)/libv8_libbase.a':Path)" />
+	<Require Condition="ANDROID" StaticLibrary="@('android/lib/@(ABI)/libv8_libplatform.a':Path)" />
+	<Require Condition="ANDROID" StaticLibrary="@('android/lib/@(ABI)/libv8_libsampler.a':Path)" />
+	<Require Condition="ANDROID" StaticLibrary="@('android/lib/@(ABI)/libv8_snapshot.a':Path)" />
 	<!-- Must be last library to avoid linker error -->
-	<Require Condition="ANDROID" StaticLibrary="@('android/lib/libv8_base.a':Path)" />
+	<Require Condition="ANDROID" StaticLibrary="@('android/lib/@(ABI)/libv8_base.a':Path)" />
```

Comment:
Didn't you just [deprecate @ABI](https://github.com/fuse-open/uno/pull/186/commits/a2cd83ebf0bb4c3853826c208cffc47b42f7963b)?

Reply:
I initially thought we needed `@(ABI)` for backwards compatibility with the current version of Uno on Fuselibs, but when I think about it using `${ANDROID_ABI}` should also work on older Uno versions, since the value will end up in CMakeLists.txt and be expanded by CMake. I'll fix.

Reply:
Ah, the former is kinda what I thought, I just expected to see it explained somewhere. Either way is fine by me, though!